CHANGELOG.md

```diff
@@ -7,6 +7,55 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 <!-- towncrier release notes start -->
 
+## [v4.0.0-rc.0]
+
+### Released 2023-09-26
+
+### Migration from v3
+
+See the [migration guide][v4_migration_guide] for details.
+
+### Breaking Changes
+
+- feat!: remove support for fluent-bit and fluentd [#3244]
+- feat!: truncate fullname after 22 characters [#3248]
+- feat(metrics)!: use otel by default [#3284]
+- feat!: use OTLP sources by default [#3297]
+- feat!(metrics): move extra processors after sumologic_schema [#3306]
+- fix(metrics)!: drop k8s.node.name attribute [#3295]
+- fix(logs)!: move JSON parsing after user-defined processors [#3281] The log body will now always be a string if accessed in extra
+  processors. Users who want to access specific fields in their parsed JSON log should explicitly call ParseJSON in their processor
+  definition.
```

Comment:
```suggestion
- fix(logs)!: move JSON parsing after user-defined processors [#3281] 

  The log body will now always be a string if accessed in extra
  processors. Users who want to access specific fields in their parsed JSON log should explicitly call ParseJSON in their processor
  definition.
```

Reply:
@swiatekm-sumo Prettier doesn't like your suggestion. The best we can do is:

```
- fix(logs)!: move JSON parsing after user-defined processors [#3281]

  The log body will now always be a string if accessed in extra processors. Users who want to access specific fields in their parsed JSON
  log should explicitly call ParseJSON in their processor definition.
```